src/App.js

```diff
@@ -0,0 +1,24 @@
+import { Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import React, { useEffect } from 'react';
+import HomePage from './pages/HomePage';
+import { fetchCountriesData } from './Redux/Countries/Countries';
+import CountriesPage from './pages/CountriesPage';
+
+const App = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchCountriesData());
+  }, []);
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you add `dispatch` to the dependency array as highlighted by React

  ![alexr-app](https://user-images.githubusercontent.com/36542986/169163826-c2cc6df6-35fa-44e9-8850-ae3213f6ba21.jpg)